engine/ladon/rego/glob/main_test.rego

```diff
@@ -0,0 +1,171 @@
+package ory.glob
```

Comment:
Great test cases, one thing I noticed (this is an issue in the other test cases too) is that it's unclear why a test case should pass or not, especially since the patterns are kinda advanced. Would it be possible to add some comments as to why a case fails or not?